charts/kubeai/values.yaml

```diff
@@ -84,6 +84,7 @@ metrics:
     vLLMPodMonitor:
       # Enable creation of PodMonitor resource that scrapes vLLM metrics endpoint.
       enabled: false
+      apiVersion: monitoring.coreos.com/v1
```

Comment:
Could you add a comment with example for what worked on Azure? Appreciate you testing this and coming up with a solution that keeps existing behavior as-is.

Reply:
I've added a comment - let me know if you would prefer alternative wording.